fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/exception/FHIRPersistenceResourceDoesntExistException.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.exception;
+
+import java.util.Collection;
+
+import com.ibm.fhir.model.resource.OperationOutcome;
+import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
+
+/**
+ * This is an exception indicating that the resource does not exist
+ */
+public class FHIRPersistenceResourceDoesntExistException extends FHIRPersistenceException {
```

Comment:
should it be in fhir-persistence (com.ibm.fhir.persistence.exception) or is it really JDBC-specific?

Reply:
also, how does it differ from FHIRPersistenceResourceNotFoundException ?

Reply:
is it actually used?

Reply:
no - removed.